CHANGELOG.rst

```diff
@@ -17,6 +17,12 @@ Added
 -----
 * Adds search script datadog_search.py, for searching Datadog monitors and dashboards.
 
+[6.0.0] - 2024-11-05
+~~~~~~~~~~~~~~~~~~~~
+Removed
+-------
+- Removes CodeOwnerMonitoringMiddleware, in favor of using new signals semt from edx-django-utils's MonitoringSupportMiddleware.
```

Comment:
```suggestion
- Removes CodeOwnerMonitoringMiddleware, in favor of using new signals sent from edx-django-utils's MonitoringSupportMiddleware.
```

Reply:
Do you also want to declare a min-dependency on edx-django-utils 7.1.0?

Reply:
Fixed, and added min dependency.